test/unit/rules/require-form-method-post-test.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+const ERROR_MESSAGE = require('../../../lib/rules/require-form-method-post').ERROR_MESSAGE;
+
+generateRuleTests({
+  name: 'require-form-method-post',
+
+  config: true,
+
+  good: [
+    '<form method="POST"></form>',
+    '<form method="post"></form>',
+
+    // dynamic values
+    '<form method="{{formMethod}}"></form>',
+    '<form method={{formMethod}}></form>',
+  ],
```

Comment:
Need to add `good` test cases to ensure false positives are avoided such as:

* `<div></div>`
* `<div method="get"></div>`